gcp/api/server.py

```diff
@@ -345,6 +346,32 @@ def DetermineVersion(self, request, context: grpc.ServicerContext):
     res = determine_version(request.query, context).result()
     return res
 
+  @ndb_context
+  @trace_filter.log_trace
+  def ImportFindings(self, request, context: grpc.ServicerContext):
+    """Return a list of `ImportFinding` for a given source."""
+    source = request.source
+    # TODO(gongh@): add source check
```

Comment:
what source check do you mean? i.e. a source name that doesn't exist? 

It would be good to make this clearer in the comment.